js/util/browser.js

```diff
@@ -9,6 +9,13 @@
  * it's used for tests.
  */
 
+var window = {
+    addEventListener: function() {},
+    removeEventListener: function() {}
+};
+
+exports.window = window;
+
```

Comment:
also needs to be added to `browser/browser.js` so that it doesn't break the non-test build